host-controller/src/main/java/org/jboss/as/host/controller/DomainModelControllerService.java

```diff
@@ -649,14 +649,15 @@ protected void boot(final BootContext context) throws ConfigurationPersistenceEx
             if (hostBootOps.isEmpty()) { // booting with empty config
                 isEmptyHost = true;
                 ok = bootEmptyConfig();
-                return;
```

Comment:
This is kind of just a code style thing, but is it correct to have bootEmptyConfig take the context as a param and create and boot with the registerModelControllerServiceInitializationBootStep(context) step, instead of booting with an empty list? Doing that and continuing to return here is cleaner looking code vs multiple checks of isEmptyHost.  (Actually if we continue to return here the 'isEmptyHost' var should be removed as it's not used in the current code.)